Source/Utils/ParticleUtils.cpp

```diff
@@ -57,12 +56,9 @@ namespace ParticleUtils
         ParticleBins bins;
         bins.build(np, ptd, cbx,
             // Pass lambda function that returns the cell index
-            [=] AMREX_GPU_DEVICE (ParticleType const & p) noexcept -> amrex::IntVect
+            [=] AMREX_GPU_DEVICE (ParticleType const & p) noexcept -> IntVect
             {
-                return IntVect{AMREX_D_DECL(
-                                   static_cast<int>((p.pos(0)-plo[0])*dxi[0] - lo.x),
-                                   static_cast<int>((p.pos(1)-plo[1])*dxi[1] - lo.y),
-                                   static_cast<int>((p.pos(2)-plo[2])*dxi[2] - lo.z))};
+                return getParticleCell(p, plo, dxi, cbx);
```

Comment:
I see most (maybe all) of the examples that involve sorting the particles per cell are still failing. Looking closer, I see that the current `development` function here calculates the cell indices with 
```
ii = (p.pos(0) - plo[0]) * dxi[0] - lo.x
```
whereas the AMReX function uses
```
ii = (p.pos(0) - plo[0]) * dxi[0] + cbx.smallend[0]
```
So the difference is that the `development` version calculates a cell index starting from 0 for each tile whereas the AMReX function calculates a global cell index for the entire domain. It's not clear to me why this breaks the code though since the binning should only care about which particles have the same cell index, not about what the value of the indices are.